packages/@vivliostyle/theme-techbook/theme.css

```diff
@@ -0,0 +1,504 @@
+  color: #f8f8f2;
+  background: none;
+  text-shadow: 0 1px rgba(0, 0, 0, 0.3);
+  font-family: Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace;
+  font-size: 1em;
+  text-align: left;
+  white-space: pre;
+  word-spacing: normal;
+  word-break: normal;
+  word-wrap: normal;
+  line-height: 1.5;
+  -moz-tab-size: 4;
+  -o-tab-size: 4;
+  tab-size: 4;
+  -webkit-hyphens: none;
+  -moz-hyphens: none;
+  -ms-hyphens: none;
+  hyphens: none;
+}
+
+/* Code blocks */
+pre[class*=language-] {
+  padding: 1em;
+  margin: 0.5em 0;
+  overflow: auto;
+  border-radius: 0.3em;
+}
+
+:not(pre) > code[class*=language-],
+pre[class*=language-] {
+  background: #272822;
+}
+
+/* Inline code */
+:not(pre) > code[class*=language-] {
+  padding: 0.1em;
+  border-radius: 0.3em;
+  white-space: normal;
+}
+
+.token.comment,
+.token.prolog,
+.token.doctype,
+.token.cdata {
+  color: slategray;
+}
+
+.token.punctuation {
+  color: #f8f8f2;
+}
+
+.namespace {
+  opacity: 0.7;
+}
+
+.token.property,
+.token.tag,
+.token.constant,
+.token.symbol,
+.token.deleted {
+  color: #f92672;
+}
+
+.token.boolean,
+.token.number {
+  color: #ae81ff;
+}
+
+.token.selector,
+.token.attr-name,
+.token.string,
+.token.char,
+.token.builtin,
+.token.inserted {
+  color: #a6e22e;
+}
+
+.token.operator,
+.token.entity,
+.token.url,
+.language-css .token.string,
+.style .token.string,
+.token.variable {
+  color: #f8f8f2;
+}
+
+.token.atrule,
+.token.attr-value,
+.token.function,
+.token.class-name {
+  color: #e6db74;
+}
+
+.token.keyword {
+  color: #66d9ef;
+}
+
+.token.regex,
+.token.important {
+  color: #fd971f;
+}
+
+.token.important,
+.token.bold {
+  font-weight: bold;
+}
+
+.token.italic {
+  font-style: italic;
+}
+
+.token.entity {
+  cursor: help;
+}
+
+code[class*=language-],
+pre[class*=language-] {
+  line-height: 1.2 !important;
+  letter-spacing: 0.03rem !important;
+  border-radius: 0.2rem !important;
+  text-shadow: none !important;
+}
+
+:not(pre) > code {
+  padding: 0.1rem 0.3rem !important;
+  color: #34495e;
+  background-color: #ecf0f1;
+}
+
+:root {
+  font-family: "Neue Frutiger World", "Verdana", "Yakumono", "body", "Hiragino Sans", sans-serif;
+  font-weight: 400;
+  line-height: 1.7;
+}
+
+h1,
+h2,
+h3 {
+  font-feature-settings: "palt", "calt";
+}
+
+h1 {
+  font-size: 2.4rem;
+  line-height: 1.2;
+}
+
+h2 {
+  font-size: 1.8rem;
+  line-height: 1.2;
+}
+
+h3 {
+  font-size: 1.4rem;
+  line-height: 1.2;
+}
+
+a {
+  text-decoration: none;
+  color: #3498db;
+}
+
+a:visited {
+  color: #9b59b6;
+}
+
+a:hover {
+  text-decoration: underline;
+  color: #e74c3c;
+}
+
+p {
+  font-feature-settings: "calt";
+  text-align: justify;
+}
+
+body {
+  counter-reset: section subsection fig tbl footnote cite;
+}
+
+a[data-ref=sec],
+a[data-ref=subsec],
+a[data-ref=fig],
+a[data-ref=tbl],
+a[data-ref=cite] {
+  text-decoration: none;
+  color: inherit;
+  margin: 0 0.15em;
+}
+
+a[data-ref=sec]::before {
+  content: target-counter(attr(href), section) "章";
```

Comment:
It seems this theme is designed for Japanese documents so I suppose it's better to rename this theme to such as `@vivliostyle/theme-techbook-ja`, otherwise it should support multi-language.

```css
html[lang=ja] a[data-ref=sec]::before {
  content: target-counter(attr(href), section) "章";
}
```

Reply:
Renamed to "theme-techbook-ja".

Reply:
I like the idea of supporting multi-language in a single theme than separating them into multiple packages.
The downside of theme with language suffix is that the number of theme packages in npm could be bloated as it continues to support a new language, and it would make search experience worsen.

How about adding `html[lang=<lang>]` selector to language specific style?

Reply:
Done: https://github.com/vivliostyle/themes/pull/7/commits/8632aa05080cb4963f585f86f5ed584e2244a894